applications/GeoMechanicsApplication/tests/submerged_construction_of_excavation/common/1_Initial_stage.json

```
@@ -0,0 +1,361 @@
+{
+    "problem_data": {
+        "problem_name": "1_Initial_stage",
+        "parallel_type": "OpenMP",
+        "echo_level": 1,
+        "start_time": -2.0,
+        "end_time": -1.0
+    },
+    "solver_settings": {
+        "time_stepping": {
+            "time_step": 1.0,
+            "max_delta_time_factor": 1000.0
+        },
+        "solver_type": "U_Pw",
+        "solution_type": "static",
+        "strategy_type": "newton_raphson",
+        "scheme_type": "newmark",
+        "model_part_name": "PorousDomain",
+        "domain_size": 2,
+        "echo_level": 1,
+        "model_import_settings": {
+            "input_type": "mdpa",
+            "input_filename": "../common/submerged_excavation_gid_project_with_interfaces"
+        },
+        "material_import_settings": {
+            "materials_filename": "../common/Linear_Elastic_Materials.json"
```

See previous suggestion

```suggestion
            "materials_filename": "Linear_Elastic_Materials.json"
```

---

See also my previous comment :-)

---

see the previous comment